bedrock/firefox/templates/firefox/whatsnew/includes/mofo-donate-cta.html

```diff
@@ -0,0 +1,36 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% if LANG.startswith('en-') %}
+  {% set donate_cta = "Donate to the Mozilla Foundation" %}
+{% elif LANG == 'fr' %}
+  {% set donate_cta = "Faire un don à la Fondation Mozilla" %}
+{% elif LANG == 'de' %}
+  {% set donate_cta = "Spende an die Mozilla Foundation" %}
+{% elif LANG == 'es-ES' %}
+  {% set donate_cta = "Donar a la Fundación Mozilla" %}
+{% elif LANG == 'it' %}
+  {% set donate_cta = "Dona alla Mozilla Foundation" %}
+{% elif LANG == 'pl' %}
+  {% set donate_cta = "Wesprzyj Fundację Mozilli darowizną" %}
+{% else %}
+  {% set donate_cta = "Donate to the Mozilla Foundation" %}
+{% endif %}
+
+<div class="wnp-donate-wrapper mzp-u-centered">
+  <a class="mzp-c-button mzp-t-secondary mzp-t-md wnp-footer-donate" href="{{ donate_url(location='whats-new-product') }}" data-link-type="button" data-link-text="Donate">
+    <span class="mzp-c-button-icon-start">
+      <svg width="16" height="16" viewBox="0 0 16 16">
```

Comment:
BTW I'm not sure how well the SVG+text render for you, I'm having better results with how line-height:2 aligns inside the button.